apps/root/src/pages/earn/components/strategies-table/components/filters/index.tsx

```diff
@@ -133,9 +140,9 @@ const Filter = <T,>({
     <StyledFilterAccordion expanded={expanded === id} onChange={handleExpandChange}>
       <AccordionSummary>
         <StyledACcordionSummaryTitle>
-          <Typography variant="bodyBold">{summaryLabel}</Typography>
+          <Typography variant="bodySmallSemibold">{summaryLabel}</Typography>
           {filteredOptions.length === 0 && (
-            <Typography variant="bodySmallBold">
+            <Typography variant="bodySmallSemibold" color={colors[mode].typography.typo4}>
```

Comment:
you can use color={({ palette: mode }) => colors[mode].typography.typo4} and it will pick it up without the need to use useThemeMode

Reply:
SIR ✨